src/main/scala/com/gu/restaurant_review_parser/ParsedRestaurantReview.scala

```diff
@@ -0,0 +1,49 @@
+package com.gu.restaurant_review_parser
+
+import java.time.OffsetDateTime
+
+case class WebAddress(value: String) extends AnyVal
```

Comment:
They're value classes (http://docs.scala-lang.org/overviews/core/value-classes.html) so that we have some type safety around all of the Strings being passed around.